src/index.html

```diff
@@ -4,14 +4,190 @@
+        <div class="group__input-block">
+          <label>
+            Surname:
+            <input
+              type="text"
+              name="surname"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="group__input-block">
+          <label>
+            Name:
+            <input
+              type="text"
+              name="name"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="group__input-block">
+          <label>
+            How old are You?
+            <input
+              type="number"
+              name="age"
+              min="1"
+              max="100"
+              value="12"
+            >
+          </label>
+        </div>
+        <div class="group__input-block">
+          <label>
+            Full date of birth:
+            <input
+              type="date"
+              name="dateOfBirth"
+            >
+          </label>
+        </div>
+        <div class="group__input-block">
+          <label>
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="termConditions"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>Registration</legend>
+        <div class="group__input-block">
+          <label>
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+        </div>
+        <div class="group__input-block">
+          <label>
+            Password:
+            <input
+              type="password"
+              name="password"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+        <div class="group__input-block">
+          <label>
+            Do you love cats?
+            <input
+              type="radio"
+              value="yes"
+              name="loveCats"
+            > Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              value="no"
+              name="loveCats"
+            > No
+          </label>
+        </div>
+        <div class="group__input-block">
+          <label>
+            What is your favorite color?
+            <input
+              type="color"
+              name="favoriteColor"
+            >
+          </label>
+        </div>
+        <div class="group__input-block">
+          <label>
+            What time do you go to bed?
+            <input
+              type="time"
+              name="timeToBed"
+              step="2"
+            >
+          </label>
+        </div>
+        <div class="group__input-block">
+          <label
+          class="cars"
+          >
```

Comment:
you can fit it in one line
```suggestion
          <label class="cars">
```